src/main/java/frc/robot/subsystems/vision/VisionSubsystem.java

```diff
@@ -65,10 +66,9 @@ && getNumberOfAprilTags(limelightNumber) <= VisionConstants.APRIL_TAG_POSITIONS.
    *
    * @param limelightNumber the number of the limelight
    */
-  public void updateLimelightPoseEstimate(int limelightNumber) {
-    // this can probably be removed after testing
+  public PoseEstimate enabledMegaTagUpdate(int limelightNumber) {
```

Comment:
nit: This method name should probably be changed -- it doesn't communicate its intent that well and it confused me reading it (maybe that's just me tho)

Reply:
i changed it to enabledPoseUpdate